app/src/main/java/org/schabi/newpipe/error/ErrorUtil.kt

```diff
@@ -0,0 +1,117 @@
+package org.schabi.newpipe.error
+
+import android.app.Activity
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.graphics.Color
+import android.os.Build
+import android.view.View
+import android.widget.Toast
+import androidx.core.app.NotificationCompat
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.google.android.material.snackbar.Snackbar
+import org.schabi.newpipe.R
+
+class ErrorUtil {
```

Comment:
I would appreciate some javadoc here, otherwise I think we are fine :)

Reply:
Yeah, I also wanted to add some javadoc but at the end I forgot. Done ;-)

Reply:
Very nice. I love it 👍